src/Avalonia.Diagnostics/Diagnostics/Convetions.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Reflection;
+using Avalonia.Controls;
+using Avalonia.VisualTree;
+
+namespace Avalonia.Diagnostics
+{
+    static class Convetions
+    {
+        public static string DefaultScreenshotsRoot =>
+             System.IO.Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.MyPictures, Environment.SpecialFolderOption.Create),
+                "Screenshots");
+
+        /// <summary>
+        /// Return the path of the screenshot folder according to the rules indicated in issue <see href="https://github.com/AvaloniaUI/Avalonia/issues/4743">GH-4743</see>
+        /// </summary>
+        public static Func<IControl, string,string> DefaultScreenshotFileNameConvention = (control,screenshotRoot)  =>
```

Comment:
Usage of this property is pretty limited, if we remove unused file system handler. 
But I guess it's fine to keep it.